packages/mobile/src/store/init/startConnection/restoreConnection/restoreConnection.saga.ts

```diff
@@ -15,6 +15,6 @@ export function* restoreConnectionSaga(): Generator {
 
   if (!isWebsocketConnected && socketIOData.dataPort !== 0) {
     console.log('WEBSOCKET', 'Restoring connection with data port: ', socketIOData.dataPort)
-    yield* put(initActions.startWebsocketConnection(socketIOData))
+    yield* putResolve(initActions.startWebsocketConnection(socketIOData))
   }
 }
```

Comment:
Since putResolve is at the end of the saga, does it have any effect if the caller of `restoreConnectionSaga` doesn't use putResolve, e.g. `putResolve(restoreConnection())`?

Reply:
To be honest, not entirely sure.